addons/website/views/snippets/snippets.xml

```diff
@@ -742,7 +745,7 @@
     <div data-js="layout_column"
         data-selector="section"
         data-target="> *:has(> .row), > .s_allow_columns"
-        data-exclude=".s_dynamic, .s_dynamic_snippet_content, .s_dynamic_snippet_title, .s_masonry_block, .s_features_grid, .s_media_list, .s_table_of_content, .s_process_steps, .s_image_gallery, .s_timeline, .s_pricelist_boxed, .s_quadrant">
+        data-exclude=".s_dynamic, .s_dynamic_snippet_content, .s_dynamic_snippet_title, .s_masonry_block, .s_features_grid, .s_media_list, .s_table_of_content, .s_process_steps, .s_image_text_overlap, .s_image_gallery, .s_timeline, .s_pricelist_boxed, .s_quadrant">
```

Comment:
Based on my previous comment, you can therefore remove it from here, to re-allow toggling to normal mode, as there is no need to block it anymore :+1: 

Note: if you want to make a snippet grid-only, you have to add it in the exclude here, but you also have to add it it the "Grid only" selectors, or else the grid mode options are not there :grimacing: So here, we cannot add new elements, and we can't change the gaps either (Spacing option) :open_mouth: :arrow_down: 
| In your PR | In masonry (how it should be) |
| ----- | ----- |
| ![image](https://github.com/user-attachments/assets/d0847fed-635b-4c4d-b99a-e7e545a91b01) | ![image](https://github.com/user-attachments/assets/52d2bc24-c2f6-4cfb-a7af-8042a9965a19) |

I will fix the other snippets :slightly_smiling_face: 
cc @stefanorigano